_posts/2015-04-20-feedhenry-push-templates.md

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
Don't you want to change the file name to newer date? Right now the blog post appears as second in the News feed, because it has older date.